spec.html

```diff
@@ -8223,17 +8256,20 @@ <h1>OrdinaryOwnPropertyKeys ( _O_ )</h1>
       </emu-clause>
     </emu-clause>
 
-    <emu-clause id="sec-objectcreate" aoid="ObjectCreate">
-      <h1>ObjectCreate ( _proto_ [ , _internalSlotsList_ ] )</h1>
-      <p>The abstract operation ObjectCreate with argument _proto_ (an object or null) is used to specify the runtime creation of new ordinary objects. The optional argument _internalSlotsList_ is a List of the names of additional internal slots that must be defined as part of the object. If the list is not provided, a new empty List is used. This abstract operation performs the following steps:</p>
+    <emu-clause id="sec-ordinaryobjectcreate" aoid="OrdinaryObjectCreate" oldids="sec-objectcreate">
+      <h1>OrdinaryObjectCreate ( _proto_ [ , _additionalInternalSlotsList_ ] )</h1>
+      <p>The abstract operation OrdinaryObjectCreate with argument _proto_ (an object or *null*) is used to specify the runtime creation of new ordinary objects. The optional argument _additionalInternalSlotsList_ is a List of the names of additional internal slots that must be defined as part of the object, beyond [[Prototype]] and [[Extensible]]. If the list is not provided, a new empty List is used. This abstract operation performs the following steps:</p>
       <emu-alg>
-        1. If _internalSlotsList_ is not present, set _internalSlotsList_ to a new empty List.
-        1. Let _obj_ be a newly created object with an internal slot for each name in _internalSlotsList_.
-        1. Set _obj_'s essential internal methods to the default ordinary object definitions specified in <emu-xref href="#sec-ordinary-object-internal-methods-and-internal-slots"></emu-xref>.
-        1. Set _obj_.[[Prototype]] to _proto_.
-        1. Set _obj_.[[Extensible]] to *true*.
-        1. Return _obj_.
+        1. Let _internalSlotsList_ be &laquo; [[Prototype]], [[Extensible]] &raquo;.
+        1. If _additionalInternalSlotsList_ is present, append each of its elements to _internalSlotsList_.
+        1. Let _O_ be ! MakeBasicObject(_internalSlotsList_).
+        1. Set _O_.[[Prototype]] to _proto_.
+        1. Return _O_.
       </emu-alg>
+
+      <emu-note>
+        <p>Although OrdinaryObjectCreate does little more than call MakeBasicObject, its use communicates the intention to create an ordinary object, and not an exotic one. Thus, it is not called by any algorithm that subsequently modifies the internal methods of the object in ways that would make the result non-ordinary. Operations that create exotic objects invoke MakeBasicObject directly.</p>
```

Comment:
"Is not called" implicitly makes a claim about other specifications. I think this should either be "is not called within this specification", or "should not be called".

Reply:
Usually when the spec talks about algorithms, it implicitly means algorithms defined in the spec. But sometimes it makes that explicit, and I agree it's worthwhile to do so here. My suggestion would be to change "Thus," to "Thus, within this specification,".

(The problem with "should not" is that it sounds like a requirement on implementations, which it isn't.)

Reply:
Ack